From e895cc8cfa323840aa5418a5070b7b1ff5976730 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Fri, 05 Oct 2018 17:43:53 +0300
Subject: [PATCH] Makefile: rename HL_RELEASE_VER => EMBD_RELEASE_VER

Since we are aligning with the general Habanalabs s/w versioning,
the HL_RELEASE_VER is no longer appropriate and even misleading name
for the embedded versions.
Thus rename to EMBD_RELEASE_VER.

There is also a direvative HL_FULL_VER variable which is generated from
the HL_RELEASE_VER. Rename it to EMBD_FULL_VER.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 73d7f5e..b583626 100644
--- a/Makefile
+++ b/Makefile
@@ -39,7 +39,7 @@
 	@echo
 	@echo 'Release:'
 	@echo '  required environment variables:'
-	@echo '    HL_RELEASE_VER     - Habanalabs release version string'
+	@echo '    EMBD_RELEASE_VER   - Habanalabs release version string'
 	@echo '  goya-release         - Prepare the release tag and push it upstream'
 	@echo '                           usually, the version string would be something'
 	@echo '                             like: x.y.z'
@@ -63,8 +63,8 @@
 
 .PHONY: hl-check-release
 hl-check-release:
-ifndef HL_RELEASE_VER
-	$(error HL_RELEASE_VER is undefined, usually looks like x.y.z)
+ifndef EMBD_RELEASE_VER
+	$(error EMBD_RELEASE_VER is undefined, usually looks like x.y.z)
 endif
 
 RELEASE_REPOS := buildroot-external preboot u-boot linux armcp tbot fw_loader btl
@@ -87,17 +87,17 @@
 
 .PHONY: gaudi-release goya-release
 gaudi-release goya-release: hl-check-release
-	$(eval HL_FULL_VER=hl-$(patsubst %-release,%,$@)-$(HL_RELEASE_VER))
+	$(eval EMBD_FULL_VER=hl-$(patsubst %-release,%,$@)-$(EMBD_RELEASE_VER))
 	$(eval RELEASE_JOB_PROD_NAME="product_name=$(patsubst %-release,%,$@)")
-	$(eval RELEASE_JOB_TAG_PTR="embedded_release_tag=$(HL_FULL_VER)")
+	$(eval RELEASE_JOB_TAG_PTR="embedded_release_tag=$(EMBD_FULL_VER)")
 	$(eval RELEASE_JOB_PARAMS=$(RELEASE_JOB_PARAMS)&$(RELEASE_JOB_PROD_NAME)&$(RELEASE_JOB_TAG_PTR))
 	@repo forall $(RELEASE_REPOS) -c \
-		git tag -afm "Release Habanalabs $(HL_FULL_VER)" $(HL_FULL_VER)
+		git tag -afm "Release Habanalabs $(EMBD_FULL_VER)" $(EMBD_FULL_VER)
 	@repo forall $(RELEASE_REPOS) -c \
 		git tag -f $(RELEASE_TAG_POINTER)
 	@-repo forall $(RELEASE_REPOS) -c git push origin :$(RELEASE_TAG_POINTER)
-	@repo forall $(RELEASE_REPOS) -c git push origin $(HL_FULL_VER) $(RELEASE_TAG_POINTER)
-	@echo "Released: $(RELEASE_BRANCH) $(HL_FULL_VER)"
+	@repo forall $(RELEASE_REPOS) -c git push origin $(EMBD_FULL_VER) $(RELEASE_TAG_POINTER)
+	@echo "Released: $(RELEASE_BRANCH) $(EMBD_FULL_VER)"
 	@wget -O - "$(RELEASE_JOB_URL)?$(RELEASE_JOB_PARAMS)"
 	@echo "Started the \"goya-release-test\" jenkins job... check your email for results."
 
